<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="mainWidjet">
   <widget class="QComboBox" name="eqvType">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Тип оборудования:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="eqvView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Вид оборудования</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>20</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Номер COM порта:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comPort">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>40</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="speedMean">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>140</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>120</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Скорость передачи</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Начать колибровку</string>
    </property>
   </widget>
   <widget class="QPushButton" name="consoleButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Закрыть консоль</string>
    </property>
   </widget>
   <widget class="QWidget" name="calibrationWidjet" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>230</y>
      <width>551</width>
      <height>281</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>541</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Следуйте указаниям с картинок. После выполнения действия нажмите кнопку продолжения колибровки</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>531</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
    <widget class="QPushButton" name="continueButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>250</y>
       <width>151</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Продолжить калибровку</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="resultsWidjet" native="true">
    <property name="geometry">
     <rect>
      <x>199</x>
      <y>229</y>
      <width>581</width>
      <height>281</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>251</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Вычесленные матрицы</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>0</y>
       <width>251</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Матрицы оборудования</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>90</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ОЗУ</string>
     </property>
    </widget>
    <widget class="QPushButton" name="transferButton">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>110</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>-&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="transferPZUButton">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>230</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Перенести в ПЗУ</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="calculatedText">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>261</width>
       <height>191</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="equipmentText">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>20</y>
       <width>261</width>
       <height>191</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>520</y>
      <width>781</width>
      <height>231</height>
     </rect>
    </property>
    <widget class="QTextBrowser" name="consoleText">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>771</width>
       <height>221</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Магнитное склонение</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>0</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Допуск отклонений:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Акселерометр</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>70</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Гироскоп</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>120</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Магнитометр</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="accelerometerAllowance">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>40</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="gyroscopeAllowance">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>90</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="magnetometerAllowance">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>140</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="magneticDeclination">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>0</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Настройки COM портов:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openCloseCom">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>90</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Открыть порт</string>
    </property>
   </widget>
   <widget class="QLabel" name="comName">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>60</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Здесь будет имя открытого порта</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSave">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuSave"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetShortcut</enum>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetShortcut</enum>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
